YouVsVirus/Assets/Scenes/Levelgethome/PlayerHouse.cs

```diff
@@ -216,8 +216,8 @@ void Update ()
     {
         if (Time.time - lastTime >= timeBetweenUpdates)
         {
-            Debug.Log("House: inside " + isPlayerInside + " close " + playerIsClose + " crruns " + aCoroutineRuns + " allowed Home: " + endlevel.isPlayerAllowedHome(player)
-                      + " at will: " + EnterAndExitAtWill);
+            //Debug.Log("House: inside " + isPlayerInside + " close " + playerIsClose + " crruns " + aCoroutineRuns + " allowed Home: " + endlevel.isPlayerAllowedHome(player)
+            //          + " at will: " + EnterAndExitAtWill);
             lastTime = Time.time;
         }
```

Comment:
If you remove the comment you can remove the whole block. Its only purpose was output.
I used it for debugging the house and you are right in that we dont need it anymore.